ldap/servers/plugins/retrocl/retrocl_po.c

```diff
@@ -691,6 +691,12 @@ retrocl_postob(Slapi_PBlock *pb, int optype)
         (void)slapi_pblock_get(pb, SLAPI_ENTRY_POST_OP, &post_entry);
         break;
     }
+    if (entry == NULL) {
+        /* To avoid coverity scan warning, but in practice, it should never happen. */
+        slapi_log_err(SLAPI_LOG_ERR, RETROCL_PLUGIN_NAME,
+                      "retrocl_postob - Operation on NULL entry\n", rc);
```

Comment:
I think it needs `%d` for the `rc`

Reply:
You are right: there is a problem, IMHO rc should better be removed  (no special rc in this test)

Reply:
fixed in https://github.com/389ds/389-ds-base/pull/6605/commits/ee5d4d0fcd27eaa1c3f272124e3501bbd1bda3db commit